### ISSUE TITLE
[task] work in progress

### ISSUE THREAD

**Issue by mckaygerhard** [h5gs/wallacepos#6]
@h5gs  @ravindragit i was checking the forks of wallace and there are some things we can do.. but you seems are so busy.. so you must define a way to merge or permit the work.. 

i was starting to work here but cos i dont have enought permission dont take any change to work.. 

i can analize and merge the others forks and discuss the continuation.. but you must delegate some functions over the hole repo or proyect

**Comment by h5gs** [comment]
Thanks for your interest and efforts. I have added you to our repo. 

In case you face any issues do let me know.

Let's get started. :)

**Comment by mckaygerhard** [comment]
ok now i will analize the others forks and take some actions, propose merge request and check if other are interested in those.. let me test those and post here a resume in few weeks

**Comment by bernd-wechner** [comment]
> ok now i will analize the others forks and take some actions, propose merge request and check if other are interested in those.. let me test those and post here a resume in few weeks

Any progress? I'm curious, as we'd love to see one of these forks up and running and taking FOSS contribs actively again. I can chip in sporadically but am seriously time poor and stretch thin as already so balked after first install attempt failed. I need something that installs and runs and I can play with before I start tinkering as the unknown up front cost to me for trying to get it to is not a goer for me (as I'm often upskilling different tech sectors as I go, who isn't? there's so much out there).

**Comment by mckaygerhard** [comment]
> > ok now i will analize the others forks and take some actions, propose merge request and check if other are interested in those.. let me test those and post here a resume in few weeks
> 
> Any progress? I'm curious, as we'd love to see one of these forks up and running and taking FOSS contribs actively again.

yes.. i passs the last 3 months **learning about accountants.. accountant is main in such systems..** also spend modules..  its hard because of the nest notes :

> I can chip in sporadically but am seriously time poor and stretch thin as already so balked after first install attempt failed. 

this is the main problem.. everybody wants to contribute but time spend is a problem.. and everybody is also busy and with lack of time.. 

including me.. that's the reason why i take so much time.. 

**These systems depend heavily on keeping accounts. Accounting is becoming a highly importat module and there is no decent one in free software, and Odoo's crap has it private** oddo is a shit cos its made in python the horny languaje for windo



>I need something that installs and runs and I can play with before I start tinkering as the unknown up front cost to me for trying to get it to is not a goer for me (as I'm often upskilling different tech sectors as I go, who isn't? there's so much out there).

as i tellyou.. Accounting is becoming a highly importat module and there is no decent one in free software, if you contributed more this being more fster to happened

the other problem is the languaje (speaking ) barrier.. English is the lang we write here but nobody like ..

**Comment by bernd-wechner** [comment]
Thanks for the update. But in all honesty, have my finger in so many pies I have to dodge some. And on this one it looks like I'm pursuing a decommission of our Tyro reader in favour of Square who provide a POS (unlike Tyro). I'll be honest, I thought h5gs was a business not a person, hence a very different time resourcing context. Still not easy and I won't dismiss the issue of having to allocate time (be it my personal time or that of staff in a business) is trivial when it comes to projects that are not obvious revenue earners and pay for themselves. So I appreciate the conundrum and thank you for goals and efforts. 

I wonder though if you aren't confusing issues a little. You mention Accounting as a module, but it's not clear what you mean. A module inside of NativePOSEngine? The human practice of accounting (as performed by accountants and bookkeepers and banks and more), a field of study (part of a university degree, like a MBA), a piece of software?

I mean, if you were looking at Odoo, I'd consider GnuCash myself (it's quite reputable) and I will be looking at Actual for our club:

https://github.com/actualbudget/actual

But WallacePOS wa just a POS, that is a software to run a kind of point of sales terminal that we could register items on, ring up a sale, and take a payment on a Tyro card reader. That has nothing to do with accounting per se, beyond th important feature of connecting eac payment with an inventory item or service. Tyro don't provide that, all they send me is a list of transactions, date, time and amount. Most stupid service I've come across. 

Square are promising better.

https://squareup.com/au/en

But I'm activating such a unit now and haven't actually tried it in the field yet. If it works I will decommission the Tyro and with it any interesting a POS for Tyro, removing one todo from my enormous list!

**Comment by mckaygerhard** [comment]
> I wonder though if you aren't confusing issues a little. You mention Accounting as a module, but it's not clear what you mean. A module inside of NativePOSEngine? The human practice of accounting (as performed by accountants and bookkeepers and banks and more), a field of study (part of a university degree, like a MBA), a piece of software?

yeah you are very clear and in right way, Accounting as a module could not be possible inside wallacepos and this is right made! must be apart, i am currently woring in the api for auth and money.. i am alone but basically we need:

**auth**> to make centralized control login, could be ldap for big things or just a simple php-auth api, this does not mean that wallace post will depends from a remote login center, but the parameters will be constantly update from

**money**> a currency rate manager and converter, with api, highly necesary for multitenant and multi enterprices companies, with various locations, this must be first to accountant, cos in latin america the rates changes too fast

and finally before POS must be the accountant module or application.. after those we cant focus on POS system.. 

i am currently working in first two.. i made a primary excersice at https://github.com/codeigniterpower/codeigniter-currency but of course need improvements.. 

those modules/apps must have something that currently nobody has: 

1. use simple tech that dont depends on networking.. nodejs is not a option then.. 
2. neither a techonology that changes with the time.. the enterprice that use the software cant depends on the 3rth party developer
